index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Tribute Page</title/>
+    <title>Tribute-Page</title/>
```

Comment:
- I think you should use good semantic tags. This makes the code to look clean and professional.  🟢📍